server/main.lua

```diff
@@ -53,7 +53,7 @@ end
 local function manageHousePlants()
     while true do
         local housePlants = MySQL.query.await('SELECT * FROM house_plants', {})
-        for i in #housePlants do
+        for i, _ in pairs(housePlants) do
```

Comment:
```suggestion
        for i = 1, #housePlants do
```

Reply:
Still won't work that way

Reply:
Why not?

Reply:
Your previous commits are different from my suggestion

Reply:
It now works, committing it now, unsure to why when I tested it with the initial value before (not in my commits). Probably an error with how I tested it to begin with.

I'll go back to feeling like a moron now.